lib/common.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 SBT_0_VERSION_PATTERN='sbt\.version=\(0\.1[1-3]\.[0-9]*\(-[a-zA-Z0-9_]*\)*\)$'
-SBT_1_VERSION_PATTERN='sbt\.version=\(1\.[0-9]*\.[0-9]*\(-[a-zA-Z0-9_]*\)*\)$'
+SBT_1_VERSION_PATTERN='sbt\.version=\(1\.[1-9]*\.[0-9]*\(-[a-zA-Z0-9_]*\)*\)$'
```

Comment:
Before v1 versions like `1.10.x` were supported, but now they won't. 

ie: Should the regex be like this?

```suggestion
SBT_1_VERSION_PATTERN='sbt\.version=\(1\.[1-9][0-9]*\.[0-9]*\(-[a-zA-Z0-9_]*\)*\)$'
```

Also it seems like some of the `*` usages should instead be `+`, otherwise versions like `1..0` would be accepted?

Reply:
Yeah, the regex is (and wasn't) great. I updated them in [541f9be](https://github.com/heroku/heroku-buildpack-scala/pull/232/commits/541f9be784fb684bae5c652903ae3f5233f564be).  👍🏻